app/src/main/res/values/strings.xml

```diff
@@ -623,7 +623,8 @@
     <string name="preferences_delete_browsing_data_button">Delete browsing data</string>
 
     <!-- Dialog message to the user asking to delete browsing data. -->
-    <string name="delete_browsing_data_prompt_message">This will delete all of your browsing data.</string>
+    <string name="delete_browsing_data_prompt_message">This will delete the data you selected.</string>
```

Comment:
CC @mheubusch

Reply:
We also need to rename strings when we change them as we don't have another process yet for updating strings across every translation. Let's wait for strings approval though.

Reply:
So should I close this PR or wait?

Reply:
@mheubusch has already approved these strings: https://github.com/mozilla-mobile/fenix/issues/4625#issuecomment-519736148

@Pheonix73 are you just able to update the 
` <string name="delete_browsing_data_prompt_message">` 
to say something like:  `<string name="delete_browsing_data_dialog_message">`? That will make it so the string is identified as new and can be translated :)

Reply:
@boek definetly. i will send a PR tomorrow.